client/lobby/SelectionTab.cpp

```diff
@@ -391,7 +392,15 @@ void SelectionTab::showPopupWindow(const Point & cursorPosition)
 		return;
 
 	if(!curItems[py]->isFolder)
-		GH.windows().createAndPushWindow<CMapOverview>(curItems[py]->getNameTranslated(), curItems[py]->fullFileURI, curItems[py]->date, ResourcePath(curItems[py]->fileURI), tabType);
+		GH.windows().createAndPushWindow<CMapOverview>(
+			curItems[py]->getNameTranslated(),
+			curItems[py]->fullFileURI,
+			tabType == ESelectionScreen::newGame && curItems[py]->mapHeader->creationDateTime ? TextOperations::getFormattedDateTimeLocal(curItems[py]->mapHeader->creationDateTime) : curItems[py]->date,
+			curItems[py]->mapHeader->author.toString() + (!curItems[py]->mapHeader->authorContact.toString().empty() ? (" <" + curItems[py]->mapHeader->authorContact.toString() + ">") : ""),
```

Comment:
Perhaps extract this into local variables to help with readability?
```cpp
if(!curItems[py]->isFolder)
{
    auto creationDateTime = ...;
    auto author = ...;
    GH.windows().createAndPushWindow<CMapOverview>( ... )
}

```

Reply:
done :)